package/CHANGELOG

```diff
@@ -128,6 +128,8 @@ Enhancements
     explicit in the topology (Issue #2468, PR #2775)
   
 Changes
+  * TPRParser now loads TPR files with `tpr_resid_from_one=True`
+    by default (PR #3152)
```

Comment:
```suggestion
  * TPRParser now loads TPR files with `tpr_resid_from_one=True` by default, 
    which starts TPR resid indexing from 1 (instead of 0 as in 1.x) (Issue #2364, PR #3152)
```